lib/WebService/MinFraud/Types.pm

```diff
@@ -84,6 +85,15 @@ sub NonNegativeNum () {
     );
 }
 
+sub SessionID () {
+    return quote_sub(
+        q{ GeoIP2::Types::_tc_fail( $_[0], 'SessionID' )
+               unless defined $_[0]
+               && ! ref $_[0]
+               && length( $_[0] ) <= 255; }
```

Comment:
We also disallow newlines and `\0`. You could probably just crib this from `SingleLineStr`.